.circleci/config.yml

```
@@ -1,23 +1,34 @@
 version: 2.1
 
 orbs:
-  python: circleci/python@0.2.1
+  python: circleci/python@2.1.1
+
 jobs:
   install_and_update_dependencies:
     docker:
-      - image: circleci/python:3.8.6-node
+      - image: cimg/python:3.8.6-node
     working_directory: ~/repo
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v2-dependencies-{{ checksum "Pipfile.lock" }}-{{ checksum "thumbs/Pipfile.lock" }}
-            - v2-dependencies-default
-      - run: &install-pipenv sudo pip install -U urllib3 cryptography==3.3.2 pipenv pip --quiet --no-input
-      - run: sudo pip uninstall -y poetry
-      - run: pipenv install --dev
-      - run: pipenv check # before save_cache so an insecure cache is never saved
+            - v3-dependencies-{{ checksum "Pipfile.lock" }}-{{ checksum "thumbs/Pipfile.lock" }}
+            - v3-dependencies-default
+      # Ensure pip and pipenv are installed
+      - run:
+          name: Install pipenv
+          command: |
+            python -m pip install --upgrade pip urllib3 cryptography==3.3.2 --quiet --no-input
```

I think explicitly installing urllib3 and pinning to an old version of cryptography here are workarounds for an issue that no longer exists.
I pushed a branch at
https://github.com/DemocracyClub/electionleaflets/commits/251-review2/
with one commit that just removes this. We can align with what you've done later in your own code
https://github.com/DemocracyClub/electionleaflets/pull/251/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R126
```suggestion
            pip install --upgrade pipenv pip --quiet --no-input
```